source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/APIControllers/Projects/GetProjectsTest.cs

```diff
@@ -267,8 +358,31 @@ public async Task GetProjects_IncludeReviewers()
             Assert.Contains(reviewer1.Id, reviewerIds);
             Assert.Contains(reviewer2.Id, reviewerIds);
             Assert.DoesNotContain(reviewer3.Id, reviewerIds);
+            var reviewer = projects.FirstOrDefault().Reviewers.FirstOrDefault();
+            Assert.Equal(reviewer1.Name, reviewer.Name);
+            Assert.Equal(reviewer1.ProjectId, project1.Id);
         }
 
+        [Fact]
+        public async Task GetProjects_IncludeAuthors()
+        {
+            BuildTestData();
+
+            var response = await Get("/api/projects?include=authors", allOrgs: true);
+            var responseString = response.Content.ToString();
+
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            var projects = await DeserializeList<Project>(response);
+
+            Assert.Equal(3, projects.Count);
+            var AuthorIds = projects.FirstOrDefault().Authors.Select(r => r.Id);
+            Assert.Equal(3, AuthorIds.Count());
+            Assert.Contains(author1.Id, AuthorIds);
+            //var c = projects.FirstOrDefault().Authors.FirstOrDefault();
```

Comment:
Probably remove this since you implemented it in a different way

Reply:
I had originally had this in the test, but for some reason it wasn't getting populated by the test but it was getting populated in real execution. I can remove it or provide better comments for whey it is commented out.  Thoughs?

Reply:
It turns out that the relationships are not Deserialized by the test helper. So remove this (and the additional stuff I had added for Reviewers